.yarnrc.yml

```diff
@@ -3,5 +3,7 @@ nodeLinker: node-modules
 plugins:
   - path: .yarn/plugins/@yarnpkg/plugin-workspace-tools.cjs
     spec: "@yarnpkg/plugin-workspace-tools"
+  - path: .yarn/plugins/@yarnpkg/plugin-interactive-tools.cjs
```

Comment:
question (non-blocking): which commands did you use and how were they helpful?

Reply:
`yarn upgrade-interactive`

If you've never used interactive upgrades then your life is incomplete:

<img width="823" alt="image" src="https://user-images.githubusercontent.com/368249/165838315-e69a3b5f-90d1-4e6e-92e3-402c318f0372.png">

Reply:
:O Sweet